back/src/business/room-name-generator.business.ts

```diff
@@ -1,22 +1,54 @@
 const baseNames = [
-  'FISTRO',
-  'PECADOR',
-  'PRADERA',
-  'NOPUEDOR',
-  'JARL',
-  'REACT',
-  'CURRY',
-  'HOOKS',
-  'KETCHUP',
-  'TURING',
-  'ALAN',
-  'CHUCK',
-  'NORRIS',
-  'DELAWARE',
-  'MALAGA',
-  'PALO',
-  'VALLEY',
-  'LEMON',
+  'Nike',
```

Comment:
Rather do not use brands (we could have legal issues), and all room names must be lowercase, let's make it all one word, how code sandbox name stuff

https://codesandbox.io/s/gracious-matsumoto-3jets